docs/content/tutorial/scheduling-your-pipeline.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: "Tutorial, part five: Scheduling your pipeline | Dagster Docs"
+description: Learn how to schedule your pipeline to run at a set frequency.
+---
+
+# Tutorial, part five: Scheduling your pipeline
+
+Now that you've written an entire pipeline in Dagster, you will need to run it regularly to keep it up to date.
+
+By the end of this part of the tutorial, you'll be able to:
+
+- Structure your project with code locations and jobs
+- Refresh your assets periodically with schedules
+- Understand your data's freshness and staleness
+
+---
+
+## Routinely refreshing your assets
+
+Data pipelines need to be run regularly to update their data. Now, you'll schedule your assets to materialize hourly. While doing so, you'll learn about two more Dagster concepts: jobs and schedules.
+
+### Defining what assets to update
+
+A _job_ lets you target a selection of assets to materialize them together as a single action. Assets can belong to multiple jobs.
+
+Your Dagster repository has a file called `tutorial/__init__.py` that is used as a top-level definition for your project. Update the code in your to add the job using the <PyObject object="define_asset_job"/> function:
+
+```python
```

Comment:
Hmm, I can't find the last message where we landed on it, but the other tutorial snippets aren't tested against `doc_snippets` because of the amount of redundant files that would get made from it. Something something like [this](https://elementl-workspace.slack.com/archives/C04QJ9M86DT/p1677183045054059?thread_ts=1677181133.269829&cid=C04QJ9M86DT)

But yes, acknowledging it's massive tech debt, hopefully punting adding diff-heavy code snippets to `docs_snippets` only until we get a new md renderer, if that's okay.

Reply:
I feel pretty strongly that code checked into the repo needs to be covered by tests.

Reply:
Is there a near term plan for an md renderer that makes this easier? What would that look like?

Reply:
Got it. I'll throw it in. By chance, got a reference to a way to patch/mock the relative import line? Currently stubbing it out by hand, but might just be overcomplicating it in my head.

```python
from . import assets
```

Reply:
Hmm good question. Could you just put a file in the same directory called assets.py?

Reply:
yeup! Did something similar, but adding boilerplate files for testing feels weird, so was asking if you new a magic way. ty!